### PR TITLE
Update the docker scripts and the versions of dependent packages

### PR DESCRIPTION
**Pull request by mjeronimo**
* The package versions now match Rolling (intend to keep in sync). This will help to keep compiles working as the latest MoveIt2 code should be building against Rolling. 

* I've added the branch info to each entry in the moveit2.repos file for future reference; makes it convenient to browse the other repos if inspecting versions (to make sure I'm looking at the right branch). 

* Also, there now seems to be no need to manually zip the .egg file from the egg-info directory, so removed that.

* Added rosidl_typesupport_fastrtps_cpp to the skip-keys list.

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

### CODE REVIEW

File: moveit2/moveit2.repos [hunk 1]
```diff
@@ -2,156 +2,195 @@ repositories:
...
+    # branch: ros2
     version: 2.1.2
   graph_msgs:
     type: git
     url: https://github.com/PickNikRobotics/graph_msgs.git
-    version: d0557ff5b19b3bdc9382e011c6613db29a69eb7b
-  ignition/ignition_cmake2_vendor:
+    # branch: ros2
+    version: 0.2.0
+  ignition_cmake2_vendor:
```
Comment (wjwwood) [inline moveit2/moveit2.repos:37]: Just curious why did you remove the `ignition` folder that these were in?

**Review by wjwwood — APPROVED** [review]
lgtm, one nitpick

File: moveit2/moveit2.repos (hunk 1 above)
Comment (mjeronimo) [reply moveit2/moveit2.repos:37]: I was comparing to the distribution.yaml for Rolling and noticed it didn't have the prefix. So, I removed it, trying to make it similar. However, it is present in the master branch of ros2.repos, so I will restore the prefix for those.

Comment (wjwwood) [reply moveit2/moveit2.repos:37]: `distribution.yaml` != a `.repos` file, I think the folder structure should be left there personally.

Comment (mjeronimo) [reply moveit2/moveit2.repos:37]: Yes, I'll update to restore those as they were (with the 'ignition' prefix).

Comment (mjeronimo) [reply moveit2/moveit2.repos]: Done.
